subiquity/server/controllers/keyboard.py

```diff
@@ -202,6 +202,11 @@ async def set_keyboard(self):
             ['setupcon', '--save', '--force', '--keyboard-only'],
             [resource_path('bin/subiquity-loadkeys')],
             ]
+        if self.app.base_model.source.current.variant == 'desktop':
```

Comment:
Instead of an explicit check on desktop, I think the key detail is if setxkbmap is around (or not).
What do you think about a shutil.which('setxkbmap') instead, and adding to the list if that is the case?

Reply:
Yes I was going to say the same thing :)

Reply:
Nice, then I don't even need to touch the test :+1: